portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```

Comment:
Do you expect this function be called multiple times? 
If yes, then you might need some code to eliminate facts duplication.

Reply:
This function only shows one fact each time the button is clicked? The original example didn't eliminate duplication, so i wasn't sure if that was needed.